chapter04/GamblersProblem.py

```diff
@@ -20,9 +20,13 @@
 # optimal policy
 policy = np.zeros(GOAL + 1)
 
-# state value
+# reward Value
+rewardValue = np.zeros(GOAL + 1)
+rewardValue[GOAL] = 1.0
+
+# state Value
 stateValue = np.zeros(GOAL + 1)
-stateValue[GOAL] = 1.0
```

Comment:
Both of codes seems to be same computationally.  But I still don't understand why this is called "asynchronous "?  I referred to Exercise 4.9 which I found not much helpful.